lib/fortnox/api/models/article.rb

```diff
@@ -0,0 +1,147 @@
+require "fortnox/api/types"
+require "fortnox/api/models/base"
+
+module Fortnox
+  module API
+    module Model
+      class Article < Fortnox::API::Model::Base
+
+        UNIQUE_ID = :article_number
+        STUB = { }.freeze
+
+        # searchable_attributes(
+        #     :article_number, :description, :ean, :manufacturer, :manufacturer_article_number,
+        #     :supplier_number
+        # )
+        #
+        # sortable_attributes(
+        #   :article_number, :disposable_quantity, :reserved_quantity, :stock_value,
+        #   :quantity_in_stock
+        # )
+        #
+        # filter(
+        #   :active
```

Comment:
Not sure why models don't list `:inactive` as well?

Reply:
It should not really be in the codebase. It was added when we worked on the filtering feature. But commented out code should not be in the repo, ever. So this should be removed. Sorry for setting a bad example there @felixyz :)

Reply:
I personally thought it was pretty helpful to have documentation for searchable/sortable but I guess the official Fortnox docs are a better source anyway.